environs/config/config.go

```diff
@@ -952,6 +972,14 @@ func (c *Config) MaxStatusHistorySizeMB() uint {
 	return uint(val)
 }
 
+// UpdateStatusHookInterval is how often to run the charm
+// update-status hook.
+func (c *Config) UpdateStatusHookInterval() time.Duration {
+	// Value has already been validated.
+	val, _ := time.ParseDuration(c.mustString(UpdateStatusHookInterval))
```

Comment:
There's an upgrade step to add the value to an existing model.
There will not be a case where the value is "" if the upgrade is done properly.